customers.code-workspace

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
@pegahnatanzi Did we want to push this for everyone else?

Reply:
@kellyayuen I think it could be helpful! so doesn't hurt. what do you think?

Reply:
Ok sounds good! I was just checking :)